websites/M/MoodlePresence/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": " .•` KIΛRΛ `•. ",
+        "id": "586905336850677760"
+    },
+    "contributors": [
+        {
+            "name": "ΛΚΘRΛ",
+            "id": "387655649934311427"
+        }
+    ],
+    "service": "Moodle",
+    "version": "1.0.0",
+    "color": "#ffbc4f",
+    "description": {
+        "en": "Moodle Rich Presence"
+    },
+    "tags": [
+        "moodle"
+    ],
+    "category": "socials",
+    "url": "moodle.com",
+    "logo": "https://cdn.discordapp.com/app-assets/821059536768335902/821099230214750279.png",
+    "thumbnail": "https://cdn.discordapp.com/app-assets/821059536768335902/821099230214750279.png",
```

Comment:
Please use Imgur

Reply:
Yeah I already sayed I will fix it later qwq